packages/hosted-credit-card-integration/src/components/HostedCreditCardComponent.test.tsx

```diff
@@ -0,0 +1,355 @@
+        </div>
+    )),
+}));
+
+jest.mock('./HostedCreditCardFieldset', () => ({
+    HostedCreditCardFieldset: ({
+        cardCodeId,
+        cardExpiryId,
+        cardNameId,
+        cardNumberId,
+        focusedFieldType,
+        additionalFields,
+    }) => (
+        <div data-test="fieldset">
+            {cardCodeId && <div data-test="cc-cvv">{cardCodeId}</div>}
+            {cardExpiryId && <div data-test="cc-expiry">{cardExpiryId}</div>}
+            {cardNameId && <div data-test="cc-name">{cardNameId}</div>}
+            {cardNumberId && <div data-test="cc-number">{cardNumberId}</div>}
+            {focusedFieldType && <div data-test="focused">{focusedFieldType}</div>}
+            {additionalFields}
+        </div>
+    ),
+}));
+
+jest.mock('./HostedCreditCardValidation', () => ({
+    HostedCreditCardValidation: ({ cardCodeId, cardNumberId, focusedFieldType }) => (
+        <div data-test="validation">
+            {cardCodeId && <div data-test="val-cc-cvv">{cardCodeId}</div>}
+            {cardNumberId && <div data-test="val-cc-number">{cardNumberId}</div>}
+            {focusedFieldType && <div data-test="val-focused">{focusedFieldType}</div>}
+        </div>
+    ),
+}));
+
+jest.mock(
+    '@bigcommerce/checkout/instrument-utils',
+    // eslint-disable-next-line  @typescript-eslint/no-unsafe-return
+    () => ({
+        ...jest.requireActual('@bigcommerce/checkout/instrument-utils'),
+        getCreditCardInputStyles: jest.fn().mockResolvedValue({}),
+        isInstrumentCardCodeRequiredSelector: () => () => true,
+        isInstrumentCardNumberRequiredSelector: () => () => true,
+        CreditCardCustomerCodeField: () => <div data-test="customer-code-field" />,
+    }),
+);
+
+jest.mock('./getHostedCreditCardValidationSchema', () => ({
+    getHostedCreditCardValidationSchema: jest.fn(() => ({})),
+}));
+jest.mock('./getHostedInstrumentValidationSchema', () => ({
+    getHostedInstrumentValidationSchema: jest.fn(() => ({})),
+}));
+
+const getDefaultProps = (overrides = {}) => ({
+    method: {
+        id: 'credit_card',
+        gateway: 'bluesnapdirect',
+        config: {
+            cardCode: true,
+            showCardHolderName: true,
+            requireCustomerCode: false,
+        },
+    },
+    checkoutService: {
+        initializePayment: jest.fn().mockResolvedValue(undefined),
+        deinitializePayment: jest.fn(),
+    },
+    checkoutState: {},
+    paymentForm: {
+        setFieldTouched: jest.fn(),
+        setFieldValue: jest.fn(),
+        setSubmitted: jest.fn(),
+        submitForm: jest.fn(),
+    },
+    language: {
+        translate: (key: string) => key,
+    },
+    onUnhandledError: jest.fn(),
+    ...overrides,
+});
+
+describe('HostedCreditCardComponent', () => {
+    it('renders HostedCreditCardFieldset with correct ids', () => {
+        render(<HostedCreditCardComponent {...getDefaultProps()} />);
+        expect(screen.getByTestId('fieldset')).toBeInTheDocument();
```

Comment:
```suggestion
        render(<HostedCreditCardComponent {...getDefaultProps()} />);
        
        expect(screen.getByTestId('fieldset')).toBeInTheDocument();
```
Kindly add an empty line before `expect` in this file.. Not sure why eslint didn't pick this up.